recorder/src/main/cpp/controller.cc

```diff
@@ -79,31 +56,45 @@ static void populate_recorder_info(recording::RecorderInfo& ri, const Configurat
             ri.set_local_time(now);
```

Comment:
It C++11 it doesn't. In C++14 it does (but gcc support falls short in this), we can't use it, because C++14 won't be available on Wheezy(we have a clang upstream build, 3.5 I believe) or Jessie.